src/spikeinterface/widgets/driftmap.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+
+import numpy as np
+
+from .base import BaseWidget, to_attr
+
+
+class DriftMapWidget(BaseWidget):
+    """
+    Plot the a drift map from a motion info dictionary.
+
+    Parameters
+    ----------
+    peaks : np.array
+        The peaks array, with dtype ("sample_index", "channel_index", "amplitude", "segment_index")
+    peak_locations : np.array
+        The peak locations, with dtype ("x", "y") or ("x", "y", "z")
+    direction : "x" or "y", default: "y"
+        The direction to display
+    segment_index : int, default: None
+        The segment index to display.
+    recording : RecordingExtractor, default: None
+        The recording extractor object (only used to get "real" times)
+    segment_index : int, default: 0
+        The segment index to display.
+    sampling_frequency : float, default: None
+        The sampling frequency (needed if recording is None)
+    depth_lim : tuple or None, default: None
+        The min and max depth to display, if None (min and max of the recording)
+    color_amplitude : bool, default: True
+        If True, the color of the scatter points is the amplitude of the peaks
+    scatter_decimate : int, default: None
+        If > 1, the scatter points are decimated
+    cmap : str, default: "inferno"
+        The colormap to use for the amplitude
+    clim : tuple or None, default: None
+        The min and max amplitude to display, if None (min and max of the amplitudes)
+    alpha : float, default: 1
+        The alpha of the scatter points
+    """
+
+    def __init__(
+        self,
+        peaks,
+        peak_locations,
+        direction="y",
+        recording=None,
+        sampling_frequency=None,
+        segment_index=None,
+        depth_lim=None,
+        color_amplitude=True,
+        scatter_decimate=None,
+        cmap="inferno",
+        clim=None,
+        alpha=1,
+        backend=None,
```

Comment:
this `backend` argument must be `None` or `matplotlib` so it is undocumented. I think it is nicer to either document and state it's value is fixed, or (probably better?) remove as an option.